test/e2e/data/deployments.go

```diff
@@ -27,6 +27,27 @@ func CreateDeploymentWithKeepPolicy(name string) *v1.AtlasDeployment {
 	return deployment
 }
 
+func CreateServerlessDeployment(name string, providerName string, regionName string) *v1.AtlasDeployment {
+	return &v1.AtlasDeployment{
+		ObjectMeta: metav1.ObjectMeta{
+			Name: name,
+		},
+		Spec: v1.AtlasDeploymentSpec{
+			Project: common.ResourceRefNamespaced{
+				Name: ProjectName,
+			},
+			ServerlessSpec: &v1.ServerlessSpec{
+				Name: name,
+				ProviderSettings: &v1.ProviderSettingsSpec{
+					ProviderName:        "SERVERLESS", // TODO: add to constants
```

Comment:
There is no real reason to add it to constants if that's the only place it's used. Please add it to constants or remove the TODO.

Reply:
Done